sentry_streams/sentry_streams/runner.py

```diff
@@ -36,24 +36,12 @@ def iterate_edges(p_graph: Pipeline, translator: RuntimeTranslator[Stream, Strea
                 if not output_steps:
                     continue
 
-                # check if the inputs are fanning out
-                if len(output_steps) > 1:
-                    pass
-
-                else:
-                    output_step_name = output_steps.pop()
-
-                    # check if the inputs are fanning in
-                    if len(p_graph.incoming_edges[output_step_name]) > 1:
-                        pass
-
-                    # 1:1 between input and output stream
-                    else:
-                        next_step: WithInput = cast(WithInput, p_graph.steps[output_step_name])
-                        print(f"Apply step: {next_step.name}")
-                        # TODO: Make the typing align with the streams being iterated through. Reconsider algorithm as needed.
-                        next_step_stream = translator.translate_step(next_step, input_stream)  # type: ignore
-                        step_streams[next_step.name] = next_step_stream
+                for output in output_steps:
```

Comment:
Thanks for the refactor here

(context) The old structure with if-elses was there just in case we needed to do something special for fan-in or fan-out